src/extension/layer-filters/src/main/java/au/org/emii/geoserver/extensions/filters/LayerFiltersService.java

```diff
@@ -133,6 +133,11 @@ public void enabledFilters(HttpServletRequest request, HttpServletResponse respo
         String workspace = request.getParameter("workspace");
         String layer = request.getParameter("layer");
 
+        if(workspace==null) {
+            workspace = layer.split(":")[0];
```

Comment:
What if there is a no 'workspace' parameter, and a 'layer' parameter with no colon in it? 

'workspace' is set to what is actually the layer name and the second split line throws an ArrayIndexOutOfBoundsException? Or am I misreading something?